src/config/routes.js

```diff
@@ -0,0 +1,3 @@
+// IMAGES
+
+export const LOGO = require('../../assets/Ask-SCORE-Logo.png');
```

Comment:
I don't get what the logo image URL is doing routes.js file.

Reply:
updated logo image URL to styles.js (config) or please tell me in which file it belongs.